### ISSUE TITLE
Disable video playback

### ISSUE THREAD

**Issue by kbumsik** [wavebox/waveboxapp#650]
* Which version of Wavebox are you using? Where is it???? I cannot find one.

* Which Operating System are you using? Manjaro Linux (Arch Linux) with up-to-date packages

Hello,
Firstly, thank you for this awesome open-source app!
Let me tell you about my exerience while using Facebook Messenger with WaveBox. You know video is automatically played when someone send a video in the chat. It is for me okay to let it play automatically.
However, it looks like it is still playing while the app is in background and constantely take CPU load.

It would be great if we can disable Video or Audio Elements manually while in background, or at least we could disable them manually.

Do you have any plan on this?

Thank you very much,
Bumsik Kim.

**Comment by nmat** [comment]
Regarding checking version you can go to Preferences -> General and scroll down to the bottom to find the version there. ;)

**Comment by Thomas101** [comment]
If you hit the cog icon in the bottom left of the screen, the current version is on the bottom right. (The latest released version is 3.14.2). There's also a `Task Monitor` button here which will give a bit more info than what `top` can on each of the processes.

![screen shot 2018-04-18 at 12 44 51](https://user-images.githubusercontent.com/103586/38930032-4d47c4dc-4306-11e8-9a1b-658d9a83d086.png)

When you say in the background, is this after you change tab to something else, or open a different app?

For changing tab, there have been some fixes that have gone out for this over the past couple of days. Might be worth checking you're on the latest version and trying again to see if you have the same behaviour. I've just tried with Facebook Messenger here and with the tab active uses 5-10% cpu. Once I switch tab to something else it drops to 0-0.2%.

As for changing to different app, this is slightly different. If the tab is left open and the app is-unfocused it will continue using the same amount of cpu. Despite the fact it's out of focus doesn't give us enough info to decide whether to throttle the tab or not, as it may still be in the monitor viewport. This is the same way other apps (chrome for example) handles going out of focus.

It would be good to know what behaviour you're seeing on the latest version to see if that matches what we see here!

**Comment by kbumsik** [comment]
Oh, I got where the version is. I was using 3.13. Thanks.
The background I was referring was to close (not quit) the app. It was using much CPU load (10%) even after close the app and the desktop environment (GNOME3 I mean) still recognized WaveBox was playing a video. That happend in version 3.13.
I upgraded to 3.14.2 today, and I am impressed. Looks like "Mini WaveBox" is active now. After closing app the background resource usaged decreased significantly. Not only it stops playing video in the background, it uses less CPU resources ever than before (like @Thomas101 saying, it is almost 0%). 
Very impresive regarding this apps is based on Electron. Great guys :)
